pages/index.ts

```diff
@@ -50,6 +50,10 @@ if (editorElement && sidebarNode) {
     })
   });
 
+  // When the user scrolls to matches, place the match in the middle of the editor.
+  const menuHeight = 47;
```

Comment:
Where does this come from /Is there a nicer way to get this value?

Reply:
I withdraw the question, as it's only related to the demo implementation.

Reply:
Haha, yeah, that was my thought 😁